### PR TITLE
Incomplete Sprint.

### PR DESCRIPTION
**Pull request by DixieKorley**

### CODE REVIEW

File: ANSWERS.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+**Vanilla JS DOM Manipulation Fu**
```
Comment (ely-alamillo) [inline ANSWERS.md:1]: Good answers to these questions.

File: components/Carousel/Carousel.js [hunk 2]
```diff
@@ -1,6 +1,29 @@
...
+
+    deselect() {
+        this.element.classList.remove("Carousel__item");
+      }
+}
+
+class CarouselArrows {
+    constructor(element) {
+        this.element = element;
+        this.element.addEventListener = ('click', event => {
```
Comment (ely-alamillo) [inline components/Carousel/Carousel.js:18]: this was a good start, heading in the right direction.

**Review by ely-alamillo — APPROVED** [review]
I noticed you didn't finish the sprint and got very little done. If there is anything we can do to help you please let us know. We are okay with sprints not being finished as long as we see the students try and demonstrate knowledge of the topics. If you would like us to help out, in anything, reach out to one of the TA's personally and we will be glad to help you as much as we can. We are here to help you be successful through your time here at Lambda School.

**Comment by DixieKorley** [comment]
Thanks alot. Yes, I had a hard time here. I wasnt sure if it was the right
direction and I wanted to understand it. The DOM, classes and functions
part of Javascript has been tricky so I'm definitely trying to study up.

**Pull request closed without merging**